documentation/docs/mcp/apify-mcp.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Apify Extension
+description: Add Apify as a Goose Extension
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CLIExtensionInstructions from '@site/src/components/CLIExtensionInstructions';
+import GooseDesktopInstaller from '@site/src/components/GooseDesktopInstaller';
+import { PanelLeft } from 'lucide-react';
+
+This tutorial covers how to add the [Apify MCP Server](https://github.com/apify/apify-mcp-server) as a Goose extension, enabling Goose to fetch context directly from Apify’s 6,000+ ready-to-use web scrapers.
+
+:::tip TLDR
+<Tabs groupId="interface">
+  <TabItem value="ui" label="Goose Desktop" default>
+  Use `Add custom extension` in Settings → Extensions to add a `Streamable HTTP` extension type with:
+  </TabItem>
+  <TabItem value="cli" label="Goose CLI">
+  Use `goose configure` to add a `Remote Extension (Streaming HTTP)` extension type with:
+  </TabItem>
+</Tabs>
+
+  **Endpoint URL**
+  ```
+  https://mcp.apify.com
+  ```
+:::
+
+## Configuration
+
+:::info
+Note that you'll need [Node.js](https://nodejs.org/) installed on your system to run this command, as it uses `npx`.
+:::
+
+<Tabs groupId="interface">
+  <TabItem value="ui" label="Goose Desktop" default>
+    1. Click the <PanelLeft className="inline" size={16} /> button in the top-left to open the sidebar
+    2. Click `Extensions`
+    3. Click `Add custom extension`
+    4. On the `Add custom extension` modal, enter the following:
+       - **Extension Name**: Apify
+       - **Type**: Streamable HTTP
+       - **Endpoint**: `https://mcp.apify.com` 
+    5. Click `+ Add` to save the header
+    6. Click `Add Extension` to save the extension
+    7. You will be taken to the OAuth screen
+    8. Navigate to the chat
+  </TabItem>
+  <TabItem value="cli" label="Goose CLI" default>
+      <CLIExtensionInstructions
+        name="apify-mcp"
+        type="http"
+        url="https://mcp.apify.com"
+        timeout={300}
+      />
+  </TabItem>
+</Tabs>
+
+## Example Usage
+
+The Apify MCP server connects Goose with [Apify](https://apify.com), a platform for web scraping and automation.  
```

Comment:
Pls let's change this:

The Apify MCP server lets you connect Goose to [Apify Store](https://apify.com/store), a marketplace of thousands of read-made tools (called _Actors_) for all kinds of web scraping, crawling, and data extraction use cases.

In this example, Goose will find an Actor for [scraping LinkedIn posts](https://apify.com/store/categories?search=linkedin) to perform a quick research.